src/base/bittorrent/session.cpp

```diff
@@ -30,6 +30,7 @@
 #include "session.h"
 
 #include <algorithm>
+#include <cstdint>
```

Comment:
Why is it needed?

Reply:
@glassez  For `std::int64_t`. https://en.cppreference.com/w/cpp/types/integer